tests/test_esphome.py

```
@@ -0,0 +1,81 @@
+        ],
+    )
+
+    assert coordinator._get_entity_id(number.DOMAIN, "t_set") == "number.current_opentherm_t_set"
+
+
+def test_get_entity_id_ignores_other_domains_for_current_esphome_unique_id() -> None:
+    coordinator = create_coordinator(
+        EntityRegistry(),
+        [
+            SimpleNamespace(
+                platform=esphome.DOMAIN,
```

Pylint (4.0.7)

[error] Instance of 'module' has no 'DOMAIN' member

(E1101)